specs/spec-2-election-ui.md

```diff
@@ -0,0 +1,200 @@
+
+Actions are broken down by actors:
+
+- Voter
+- Candidate
+- Watchdog (no IAH credentials)
+
+These user stories illustrate the needs and desires of the actors rather than perscribing how the actors will achieve their goals.
+
+### Voter
+
+#### Outside of Election UI but still in BOS
+
+- I want to be notified when the election opens and notified when the election is about to close directly on BOS [notification widget].
+- I want to know about upcoming elections and their dates. 
+
+#### Viewing a Live Election
+
+- I want to see all the candidates for each congressional body, so I know my options.
+- I want to see current voting results, so I know everything is happening on chain.
+- I want to see how many candidates I can vote for per congressional body, so I know how to select candidates.
+- I want to see my past votes casted for each congressional body, so I know who I voted for.
+- I want to understand the time frame I'm looking at defined both in terms of Congressional # and year.
+- I want to see the results of an election after it was over with transaction history, so there is accountability after the election.
+
+#### Voting in a Live Election
+
+- I want a cart of candidates, so I can organize who I am voting for before I cast a vote on chain.
+- I want to be able to vote on each Congressional body one at a time so I don’t have to make the entire decision at once.
+- I want to be able to toggle between Congressional bodies so I can more easily choose candidates and my vote status.
+- I want to see how many votes I have cast and have left for each congressional body, to better allocate my votes and understand the quadratic voting process.
+- I want to learn about quadratic voting because it's new to me.
+- I want to easily increment and decrement the votes I've assigned to each canidate. 
+- I want to know when I've used up all of my votes so I don't case more votes than I have (Disabled UI, toast notification)
+- I want to save my progress for voting/cart, so I can resume + calculate quadratic voting strategy at a later vote without having to submit the whole decision.
+- I want to see time until the next Congressional election after a vote is over, so I can plan to run for upcoming congressional elections.
+- I want to see past election results (even if its first vote - show no past election), so I can gauge the electorate’s sentiment.
+
+#### Viewing Election Candidates
+
+- I want to see a candidate profile (Near Social) and the platform they used to nominate themselves (nomination mechism).
+- I want to see a candidate's past congressional history on profile (past roles, proposals, and complaints), so I can audit their past performance.
+- I want to see the profile + attestation (queried from the nomination process) for a candidate so I can see social proof for why someone was elected.
+- I want to see what DAOs a candidate is in even if they are not part of NDC, in order to understand what communities they are a part of.
```

Comment:
We will need to specify it. How we will keep track of this? Are we only going to look at Astro DAO (only contract deployed by the factory), or other contracts as well?